clang/lib/Frontend/CompilerInstance.cpp

```diff
@@ -756,6 +756,14 @@ void CompilerInstance::createSema(TranslationUnitKind TUKind,
     TheSema->addExternalSource(ExternalSemaSrc.get());
     ExternalSemaSrc->InitializeSema(*TheSema);
   }
+
+  // If we're building a module and are supposed to load API notes,
+  // notify the API notes manager.
+  if (auto currentModule = getPreprocessor().getCurrentModule()) {
```

Comment:
Does `getCurrentModule()` return a pointer?  The type is not clear, and this might be a copy.  I think that explicitly spelling out the type would be helpful.

Reply:
Alright, done